Formula/kyoto-tycoon.rb

```diff
@@ -12,10 +12,21 @@ class KyotoTycoon < Formula
     sha256 "e75c60a4417bc00d04e1f24241320329f01b0d3076de2585e92375b12c4ef31d" => :sierra
   end
 
+  depends_on "lua" => :build
   depends_on "kyoto-cabinet"
-  depends_on "lua"
+  depends_on "zlib"
 
-  patch :DATA
+  # Build patch (submitted upstream to info@fallabs.com)
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/955ce09/kyoto-tycoon/0.9.56.patch"
+    sha256 "7a5efe02a38e3f5c96fd5faa81d91bdd2c1d2ffeb8c3af52878af4a2eab3d830"
+  end
+
+  # Homebrew-specific patch to support testing with ephemeral ports (often returned by `free_port`)
```

Comment:
Is there any reason why it would not be useful to submit this upstream?

Reply:
My bad. Mail to upstream sent.